clang/include/clang/Basic/Attr.td

```diff
@@ -4613,14 +4613,13 @@ def HLSLGroupSharedAddressSpace : TypeAttr {
   let Documentation = [HLSLGroupSharedAddressSpaceDocs];
 }
 
-def HLSLParamModifier : TypeAttr {
+def HLSLParamModifier : ParameterABIAttr {
```

Comment:
as an aside: what's the plan for / state of "uniform".  It's mentioned in the docs at least.

Reply:
DXC currently parses but ignores the `uniform` keyword. We have an issue to just officially remove it from the language in 202x since it doesn't do anything, so my plan is to not implement it in Clang at all (https://github.com/microsoft/hlsl-specs/issues/135)